packages/headless/src/invoices.ts

```diff
@@ -15,6 +15,27 @@ import { MutationKey, QueryKey } from "@/constants";
 type OnSuccess = Void;
 type OnError = (error: Error) => void;
 
+type useFindInvoicesProps = {
+  query: UseInfiniteQueryResult<
+    InfiniteData<Paginated<IInvoice.Self>, unknown>,
+    Error
+  >;
+  list: IInvoice.Self[] | null;
+  more: () => void;
+};
+
+export function useFindInvoices({
+  pageParam,
+}: {
+  pageParam: number;
+}): useFindInvoicesProps {
+  const atlas = useAtlas();
+  const findInvoices = useCallback(async () => {
+    return await atlas.invoice.find({ page: pageParam });
+  }, []);
```

Comment:
You are missing a dependency in you `useCallback`. it is `atlas.invoices` to get some help you can install `eslint` extension.